tests/test_matrix.py

```diff
@@ -173,3 +189,34 @@ def test_multiplyMatrix():
     # assert matPro.__str__() == ""
     """
     pass
+
+
+#################
+# matrix.structure #
+#################
+
+
+def test_transposeMat():
+
+    mat = getTokens("[1, 2; \
+                    3,  4]")
+    matTranspose = Matrix.transposeMat(mat)
```

Comment:
Use `mat.transpose()` instead of `Matrix.transposeMat(mat)`. Looks cleaner. Same in line 209 as well.
```suggestion
    matTranspose = mat.transpose()
```

Reply:
OOPs xD